src/nodes.coffee

```diff
@@ -1518,12 +1531,16 @@ exports.Index = class Index extends Base
   compileToFragments: (o) ->
     [].concat @makeCode("["), @index.compileToFragments(o, LEVEL_PAREN), @makeCode("]")
 
-  ast: ->
-    @index.ast()
-
   shouldCache: ->
     @index.shouldCache()
 
+  ast: ->
+    # Babel doesn’t have an AST node for `Index`, but rather just includes
```

Comment:
Same idea as above. I'd personally find it pretty obvious what's going on to just see the body of the function as `@index.ast()`, so I don't think the comment would need any explanation of the mechanics of what's going on. As far as the "why", again if that's even necessary/helpful, you could be specific that an `Index` will correspond to `property` on a `MemberExpression` where `computed: true`